base/exports.jl

```diff
@@ -1149,6 +1149,7 @@ export
     ntoh,
     open,
     pipe,
+    Pipe,
```

Comment:
So what does the `pipe` function do now then? Either connect two commands via a pipe or construct a pipe object?